### PR TITLE
Bug 1457833- New XCUITests download manager feature

### PR DESCRIPTION
**Pull request by isabelrios**
This PR is to add XCUITests to cover the new feature which allows user to download files

### CODE REVIEW

File: XCUITests/DownloadFilesTests.swift [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        navigator.openURL(testURL)
+        waitUntilPageLoad()
+
+        app.webViews.staticTexts[fileName].tap()
+        app.webViews.buttons["Download"].tap()
+        waitforExistence(app.tables["Context Menu"])
+        app.tables["Context Menu"].cells["download"].tap()
+    }
+
+    func testDownloadMoreThanOneFile() {
```
Comment (npark-mozilla) [inline XCUITests/DownloadFilesTests.swift]: perhaps it could be used in the future if this takes a parameter and download x number of files?

Comment (isabelrios) [reply XCUITests/DownloadFilesTests.swift]: So you mean to modify the `private func downloadFile(fileName: testFileName)` function to something like:
`private func downloadFileXtimes(fileName: String, numberOfDownloads: Int)`?

Comment (npark-mozilla) [reply XCUITests/DownloadFilesTests.swift]: yes, that would be more useful in the long run.

Comment (isabelrios) [reply XCUITests/DownloadFilesTests.swift]: 👍

File: XCUITests/DownloadFilesTests.swift [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        app.tables.cells.staticTexts[testFileName].press(forDuration: 2)
+        waitforExistence(app.otherElements["ActivityListView"])
+        if iPad() {
+            app.otherElements["PopoverDismissRegion"].tap()
+        } else {
+            app.buttons["Cancel"].tap()
+        }
+     }
+
+    private func downloadFile(fileName: String, numberOfDownlowds: Int) {
```
Comment (isabelrios) [inline XCUITests/DownloadFilesTests.swift]: @npark-mozilla Please let me know your view on this, if this is now more similar to what you commented. Thanks!

**Pull request merged**